### PR TITLE
fix(#3233): fix rendering 0 when there are no errors.

### LINKED ISSUE #3233: Rendering `0` for fields when there are no errors after adding a custom validator
### Prerequisites

- [X] I have searched the existing issues
- [X] I understand that providing a [SSCCE](http://sscce.org/) example is tremendously useful to the maintainers.
- [X] I have read the [documentation](https://react-jsonschema-form.readthedocs.io/)
- [X] Ideally, I'm providing a [sample JSFiddle](https://jsfiddle.net/n1k0/f2y3fq7L/6/) or a [shared playground link](https://rjsf-team.github.io/react-jsonschema-form/) demonstrating the issue.

### What theme are you using?

antd

### Version

5.x

### Current Behavior

After adding a custom validator, there will render a `0` error message when there are no errors.

image

### Expected Behavior

No error messages or otherwise being rendered when there are no errors.

### Steps To Reproduce

Just add a custom validator function and return the `props.errors` directly.
This is a sample codesandbox.
https://codesandbox.io/s/condescending-field-616vmc
Need to input something to trigger validate.

### Environment

```markdown
- OS:MacOS 12.6.1
- Node:v16.17.0
- npm:8.19.2
```

### Anything else?

Nothing.

### PR DESCRIPTION
**Pull request by suyanhanx**
### Reasons for making this change

fixes #3233

### Checklist

* [ ] **I'm updating documentation**
  - [ ] I've [checked the rendering](https://react-jsonschema-form.readthedocs.io/en/latest/#contributing) of the Markdown text I've added
* [ ] **I'm adding or updating code**
  - [ ] I've added and/or updated tests. I've run `npm run test:update` to update snapshots, if needed.
  - [ ] I've updated [docs](https://react-jsonschema-form.readthedocs.io/) if needed
  - [ ] I've updated the [changelog](https://github.com/rjsf-team/react-jsonschema-form/blob/main/CHANGELOG.md) with a description of the PR
* [ ] **I'm adding a new feature**
  - [ ] I've updated the playground with an example use of the feature

### CODE REVIEW

File: packages/antd/src/templates/FieldTemplate/index.tsx [hunk 1]
```diff
@@ -69,7 +69,7 @@ const FieldTemplate = ({
           colon={colon}
           extra={rawDescription && description}
           hasFeedback={schema.type !== "array" && schema.type !== "object"}
-          help={(!!rawHelp && help) || (rawErrors?.length && errors)}
+          help={(!!rawHelp && help) || (rawErrors?.length ? errors : undefined)}
```
Comment (heath-freenome) [inline packages/antd/src/templates/FieldTemplate/index.tsx:72]: Can you add a test that verifies this fix and also update the `CHANGELOG.md`

Comment (heath-freenome) [reply packages/antd/src/templates/FieldTemplate/index.tsx:72]: Actually, It looks like the test snapshot just needs to be updated

**Comment by heath-freenome** [comment]
@suyanhanx We are still waiting for a couple of changes so that we can merge your fix

**Comment by suyanhanx** [comment]
@heath-freenome Sorry to keep you waiting for so long. I've updated the snapshot and changelog.

**Comment by heath-freenome** [comment]
@suyanhanx How did you update the snapshot as yours seem to have broken the build. You can simply use `npm run test:update` to do this.

**Comment by suyanhanx** [comment]
@heath-freenome Well, I did this manually. Now I have used the command you suggested to update snapshots. Thanks for your suggestion.

File: CHANGELOG.md [hunk 2]
```diff
@@ -34,6 +34,9 @@ should change the heading of the (upcoming) version to include a major version b
 - For JSON Schemas with `$id`s, use a pre-compiled Ajv validation function when available.
 - No longer fail to validate inner schemas with `$id`s, fixing [#2821](https://github.com/rjsf-team/react-jsonschema-form/issues/2181).
 
+## @rjsf/antd
+- No longer render extra 0 for array without errors, fixing [#3233](https://github.com/rjsf-team/react-jsonschema-form/issues/3233)
+
```
Comment (heath-freenome) [inline CHANGELOG.md:39]: Can you move this up higher in the file under the `beta.14` section

**Review by heath-freenome — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (suyanhanx) [reply CHANGELOG.md:39]: I'm sorry I just saw it now. I'd be happy to go and fix it, but it's a pity you've already merged. I think I'll check the notifications more often in the future.

Comment (heath-freenome) [reply CHANGELOG.md:39]: No worries, I was putting in a fix of my own and fixed up the `CHANGELOG.md` as part of that change

Comment (suyanhanx) [reply CHANGELOG.md:39]: I see it. Thanks.

**Pull request merged**